python/tests/test_dataframe.py

```diff
@@ -0,0 +1,998 @@
+#
+#  Copyright 2015-2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+import unittest
+import os
+
+import ts.flint.dataframe
+import pandas as pd
+import numpy  as np
+import pandas.util.testing as pdt
+
+from spark_test_case import SparkTestCase
+from pdf import make_pdf
+from test_data import FORECAST_DATA, PRICE_DATA, VOL_DATA, VOL2_DATA, VOL3_DATA, INTERVALS_DATA
+
+BASE = eval(os.getenv('BASE_CLASS', 'SparkTestCase'))
+
+class TestDataframe(BASE):
+    ''' Tests for DataFrame '''
+
+    def forecast(self):
```

Comment:
I prefer to move these in a base class as well, so other test cases can inherit and have data automatically.